app-emulation/rkt/rkt-9999.ebuild

```diff
@@ -32,7 +32,7 @@ DEPEND=">=dev-lang/go-1.2
 RDEPEND=""
```

Comment:
We are actually using go 1.3 in the SDK, but if that is a strict requirement it is worth documenting in the ebuild.